src/postgres/tree/PostgresFunctionsTreeItem.ts

```diff
@@ -32,13 +36,59 @@ export class PostgresFunctionsTreeItem extends AzureParentTreeItem<ISubscription
     }
 
     public async loadMoreChildrenImpl(): Promise<PostgresFunctionTreeItem[]> {
+        const rows: IPostgresProceduresQueryRow[] = await this.listFunctions();
+        this._functionsAndSchemas = {};
```

Comment:
Do you want to clear out `this._functionsAndSchemas` every time you loadMoreChildren? I figured we only need to clear the object when we do a refresh

Reply:
If we didn't clear it, would we only populate new entries in `functionsAndSchemas`? I feel like we would still need to get all the rows from the server to see if anything changed

Reply:
I was thinking a refresh would be able to take care of that, but load children gets called when you expand/refresh anyway (I think?) so it's probably better to do it here anyway.